app/src/main/java/no/usn/mob3000/domain/viewmodel/socials/ProfileViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package no.usn.mob3000.domain.viewmodel.socials
+
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateMapOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+import no.usn.mob3000.domain.model.social.FriendData
+import no.usn.mob3000.domain.usecase.social.FetchFriendsUseCase
+/**
+* @author Hussein Abdul-Ameer
+* created on 05.11.2024
```

Comment:
Remember to use GitHub usernames and ISO8601 conformed dates. Additionally, this has been a problem in every one: Remember to actually use the KDoc. They're there to describe the class, function, object, etc. If they are empty, they serve little to no purpose.